aiida_vasp/calcs/tests/test_vasp.py

```diff
@@ -32,61 +32,6 @@ def test_write_incar(fresh_aiida_env, vasp_calc_and_ref):
             assert result_incar_fo.read() == reference['incar']
 
 
-@STRUCTURE_TYPES
-def test_write_poscar(fresh_aiida_env, vasp_calc_and_ref, vasp_structure_poscar):
```

Comment:
Concerning all the `test_write_X` tests here, I get that the correctness of output file writing is the responsibility of the file parser (and ultimately of `parsevasp`) now and should be tested there. I could agree to leave these methods untested and add tests only if needed to avoid regression in case they get broken.

However, if someone would say that it would raise their trust in the plugin if there was at least a test that this function calls the file parser and writes a file, I would think it a small price to pay to add one.

Reply:
Parsevasp already have a rather extensive test set. In fact, most things have a test. There are some remaining, as always. I think the coverage is above 70% at this stage. I left these particular tests since the core of the vasp plugin uses these directly so I thought it might be nice to have them around. In case something in the interface gets funky. In fact it calls a write, which gets an override etc. So just testing it in parsevasp would not necessarily be enough. I will leave them for now.